README.md

```diff
@@ -1256,6 +1256,20 @@ officially supported LSP server for Ruff.
 However, Ruff is also available as part of the [coc-pyright](https://github.com/fannheyward/coc-pyright)
 extension for `coc.nvim`.
 
+<details>
+<summary>Ruff works with <a href="https://github.com/dense-analysis/ale"><code>ale</code></a>.</summary>
```

Comment:
I'd suggest to change the wording to:

> with the ALE plugin for (Neo)Vim

since readers might not be familiar with ALE and this added context helps them decide if they may want to check it out